hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/NetworkGetAccountDetailsHandler.java

```diff
@@ -98,6 +100,7 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
         final var accountStore = context.createStore(ReadableAccountStore.class);
         final var accountMetadata = accountStore.getAccountById(op.accountIdOrThrow());
```

Comment:
nit : It is not `accountMetadata` . Instaed it is `account. It was metadata before changing all stores to return account object 🙂

Reply:
Whoops, you're right!

Reply:
Done